feat: Added AddAdditionalAttributesAfterMfa filter (#17)
CESNET/simplesamlphp-module-authswitcher: README.md

When the attribute is not empty, single factor authentication is not considered. Therefore, when an SP requests `SFA` or `PasswordProtectedTransport` specifically, the authentication will fail.

> ## Add additional attributes after MFA is performed

Reviewer: ```suggestion
## Add additional attributes when MFA is performed
```

> To add attributes, there is a filter called `AddAdditionalAttributesAfterMfa`.

Reviewer: ```suggestion
To add attributes only if MFA was performed, you can use a filter called `AddAdditionalAttributesAfterMfa`.
```

> `AddAdditionalAttributesAfterMfa` does not work if called before the `SwitchAuth` filter.

Reviewer: ```suggestion
`AddAdditionalAttributesAfterMfa` needs to run after the `SwitchAuth` filter.
```
